dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -759,27 +759,21 @@ def predict(self, data, batch_size=32, show_confidences=False,
             allocation_index += num_samples_in_batch
 
         # Convert predictions, confidences to lists from numpy
-        predictions = predictions.tolist()
+        r_predictions = []
+        r_confidences = []
```

Comment:
why changing the name with the prefix `r`?

Reply:
Because the predictions and confidence arrays don't change.

The r_ is representing "return".